native/acter/src/api/room.rs

```diff
@@ -1757,7 +1757,7 @@ impl Room {
                     .get_member(&my_id)
                     .await?
                     .context("Unable to find me in room")?;
-                if !member.can_redact() {
+                if !member.can_redact_own() {
```

Comment:
same here we first need to figure out whether this is our own and then check accordingly ... but that means knowing the original author_id I suppose ... so I guess we have to alter the API to require that is also provided.

Or change the api-docs to say that this isn't checking for the permission and permission thus must have been checked before the call. Which I'd consider fine, too.

Reply:
Sorry for my bad.
Fixed by https://github.com/acterglobal/a3/pull/1311/commits/1989a5e1c1f64d6b0d0d0761ed32291247856c59